index.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  extend: 'apostrophe-widgets',
+  label: 'Address',
+  addFields: [
+    {
+      name: 'lineOne',
+      label: 'Address Line 1',
+      type: 'string'
```

Comment:
should `lineOne` be required?

Would `street1` be clearer?

Reply:
I do think it makes sense to require line one, see 163e5f1.

The `lineOne` and `lineTwo` naming convention was accounting for the case where this might be a building name, an apartment number or floor or something non-street related.